src/components/input/bl-input.css

```diff
@@ -44,6 +45,14 @@
   --icon-size: var(--bl-font-body-text-3-line-height);
 }
 
+.wrapper:has(input:autofill) {
+  background-color: var(--autofill-bg-color);
+}
+
+.wrapper:has(input:-webkit-autofill) {
+  background-color: var(--autofill-bg-color);
+}
```

Comment:
```suggestion
.wrapper:has(input:autofill),
.wrapper:has(input:-webkit-autofill) {
  background-color: var(--autofill-bg-color);
}
```

Reply:
It doesn't work that way, so I seperated them.

Reply:
How could that be possible? 🤔 Weird.. I'll check the reason.

Reply:
The :has pseudo-class is not supported by Firefox and by recent (not so old) versions by chrome and safari. We may think of another ways to implement these.

https://caniuse.com/css-has

Reply:
You're right, the browser support is not enough yet to implement this. I'm open to suggestions for alternative solutions.

Reply:
I added my comments on browser support earlier:
> Because of [browser support for :has selector](https://caniuse.com/?search=%3Ahas) not perfect yet, I see this will not make the experience of some users about auto filling very good. But I also see that, trying to cover older browsers will need some work on JS and this will add performance costs for every user. So I think it's ok to keep this in that way for now.
> 
> https://github.com/Trendyol/baklava/pull/412#discussion_r1102737194

I have some ideas to make this without `:has` selector but I don't see any way that doesn't effect the users who has a modern browser (85% of all global users) in terms of performance.